tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS, ALL_V2_ENVIRONMENTS
+from metaworld.policies import *
+from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
+
+test_data_v1 = []
+
+test_data_v2 = [
+    # name,      policy,      success rate,   environment kwargs
+    ['push-wall-v2', SawyerPushWallV2Policy(), 0.8, {}]
+]
+
+for row in test_data_v1:
+    row[-1] = ALL_ENVIRONMENTS[row[0]](random_init=True, **row[-1])
+
+for row in test_data_v2:
+    row[-1] = ALL_V2_ENVIRONMENTS[row[0]](random_init=True, **row[-1])
```

Comment:
If `test_data` is combined, `ALL_ENVIRONMENTS` and `ALL_V2_ENVIRONMENTS` will need to be union'd: `ALL_ENVIRONMENTS.update(ALL_V2_ENVIRONMENTS)` to join

Reply:
Ok that works. I didn't want to add the V2 environments to the ALL_ENVIRONMENT dict, so I kept them as separate loops, but this is good too

Reply:
Good point. Maybe create a new (empty) dictionary and update it with the entries from both `ALL_ENVIRONMENTS` and `ALL_ENVIRONMENTS_V2`? That way we still have access to the original dicts. If you can think of another way, that's fine too.